src/components/SimpleTable.tsx

```diff
@@ -125,49 +31,111 @@ export interface IProps {
-        <HeaderCell key={idx}>{column}</HeaderCell>
-      ))}
-    </Header>
-    {hasError && (
-      <Row>
-        <Cell>
-          <Translation id="page-stablecoins-table-error" />
-        </Cell>
-      </Row>
-    )}
-    {!hasError && content.length === 0 && (
-      <Row>
-        <Cell>
-          <Translation id="page-stablecoins-table-loading" />
-        </Cell>
-      </Row>
-    )}
-    {content.map((row, idx) => {
-      const { name, marketCap, image, type, url } = row
-      const rowContent = (
-        <>
-          <Cell>
-            {image && <StyledImage src={image} alt="" />}
-            {name}
-          </Cell>
-          <Cell>{marketCap}</Cell>
-          <Cell>{type}</Cell>
-        </>
-      )
+const SimpleTable: React.FC<IProps> = ({ columns, content, hasError }) => {
+  const [textColor] = useToken("colors", ["text"])
+
+  return (
+    <Table
+      variant="unstyled"
+      my={8}
+      borderRadius="sm"
+      border={`1px solid ${textColor}`}
+      bg="background"
+      mb={8}
+      minW="720px"
+    >
+      <Thead bg="ednBackground" color="text200">
+        <Tr boxShadow={`0 1px 1px ${textColor}`} mb="1px">
+          {columns.map((column, idx) => (
+            <Th
+              key={idx}
+              py={5}
+              fontSize="md"
+              fontWeight="normal"
+              letterSpacing="normal"
+            >
+              {column}
+            </Th>
+          ))}
 
-      return url ? (
-        <LinkRow key={idx} to={url}>
-          {rowContent}
-        </LinkRow>
-      ) : (
-        <Row key={idx}>{rowContent}</Row>
-      )
-    })}
-  </Table>
-)
+          {content && content[0]?.url && (
+            <Th p={5} fontSize="md" fontWeight="normal" textAlign="end">
+              ↗
+            </Th>
+          )}
+        </Tr>
+      </Thead>
+      <Tbody>
+        {hasError && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-error" />
+            </Td>
+          </Tr>
+        )}
+        {!hasError && content.length === 0 && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-loading" />
+            </Td>
+          </Tr>
+        )}
 
+        {content.map(({ name, marketCap, image, type, url }, idx) => (
+          <LinkBox
+            as={Tr}
+            display="table-row"
+            href={url}
+            key={idx}
+            color="text"
+            textDecoration="none"
+            boxShadow={`0 1px 1px ${textColor}`}
+            _hover={{
+              textDecoration: "none",
+              boxShadow: `0 0 1px ${textColor}`,
+              bg: "primary200",
+              color: "black300",
+              cursor: url ? "pointer" : "default",
+            }}
+            _focus={{
+              boxShadow: `0 0 1px ${textColor}`,
+              bg: "primary200",
+              color: "black300",
+            }}
+          >
```

Comment:
A couple of props should be removed here, as they are ineffective or not needed.
`display`, `href`, and `textDecoration`
```suggestion
          <LinkBox
            as={Tr}
            key={idx}
            color="text"
            boxShadow={`0 1px 1px ${textColor}`}
            _hover={{
              textDecoration: "none",
              boxShadow: `0 0 1px ${textColor}`,
              bg: "primary200",
              color: "black300",
              cursor: url ? "pointer" : "default",
            }}
            _focus={{
              boxShadow: `0 0 1px ${textColor}`,
              bg: "primary200",
              color: "black300",
            }}
          >
```

Reply:
Thanks for the suggestions @TylerAPfledderer! 
I think we can also remove the `textDecoration` props from the hover style.